### PR TITLE
Update per Netflix whitepaper

### LINKED ISSUE #3: Is #textEmphasis necessary, or is #textEmphasis-minimal sufficient?
Is there a use case in subtitles/captions for anything beyond `#textEmphasis-minimal`?

### LINKED ISSUE #8: Update per Netflix input document
See https://lists.w3.org/Archives/Public/public-tt/2017Nov/0007.html

### PR DESCRIPTION
**Pull request by palemieux**
Closes #8 
Closes #3

Also refactored tts:textShadow per https://www.w3.org/AudioVideo/TT/tracker/issues/234

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -224,6 +229,21 @@
       <p>To simplify style authoring, [[!ttml-imsc1.1]] Text Profile SHALL support the <code>#initial</code> feature specified in
       [[!TTML2]].</p>
     </section>
+
+    <section>
+      <h3><code>#textShadow</code></h3>
+
+      <p>To support the edge styles required by the FCC, [[!ttml-imsc1.1]] Text Profile SHALL support the <code>#textShadow</code>
```
Comment (nigelmegitt) [inline index.html:242]: Is this genuinely a requirement of FCC that is not currently met with textOutline? I thought that had already been deemed sufficient.

**Review by nigelmegitt — COMMENTED** [review]
Question asked about FCC requirements.

File: index.html (hunk 1 above)
Comment (palemieux) [reply index.html:242]: @mikedo What say you?

Comment (andreastai) [reply index.html:242]: There are some issues with tts:textOutline but I can not remember what they have been. Possibly we can look at some examples.

**Review by cconcolato — COMMENTED** [review]
rubyAlign should be 'auto' only if it means center, otherwise 'center' should also be allowed. The white paper was not so clear.

rubyPosition 'auto' would be fine if defined to map 'outside'

I think it misses writing-mode.

**Comment by palemieux** [comment]
>I think it misses writing-mode.

@cconcolato `writingMode` is already allowed in IMSC 1.1

**Comment by palemieux** [comment]
> rubyPosition 'auto' would be fine if defined to map 'outside'

@cconcolato Why not constrain `rubyPosition="outside"` then? Or do you mean that both `auto` and `outside` need to be allowed.

**Comment by palemieux** [comment]
> rubyAlign should be 'auto' only if it means center, otherwise 'center' should also be allowed. The white paper was not so clear.

@cconcolato Should we constrain `rubyAlign="center"` then? Or do you mean that both `auto` and `center` need to be allowed?

**Comment by cconcolato** [comment]
For rubyPosition and rubyAlign, we care for the default behavior, whatever the keyword value is.

**Review by cconcolato — APPROVED** [review]
LGTM

File: index.html (hunk 1 above)
Comment (palemieux) [reply index.html:242]: > There are some issues with tts:textOutline but I can not remember what they have been. Possibly we can look at some examples.
 
Per the joint CSS/TTWG meeting, text-outline is not well-supported in UAs today. No one disputed the value of the feature however -- as I recall.

Comment (nigelmegitt) [reply index.html:242]: Yes, that's what I recall - the problem with `tts:textOutline` is not that it is a poor fit for the requirement, but that it is not widely implemented. If it _were_ implemented, presumably support for `#textShadow` would no longer be required to support the edge styles required by FCC?

**Comment by mikedo** [comment]
> @nigelmegitt Is this genuinely a requirement of FCC that is not currently met with textOutline? I thought that had already been deemed sufficient.

textShadow better matches the FCC requirements than textOutline, but I need time to study whether textShadow alone is sufficient.

**Pull request merged**